datasets/opus_ubuntu/opus_ubuntu.py

```diff
@@ -0,0 +1,133 @@
+# coding=utf-8
+# Copyright 2020 HuggingFace Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+import os
+
+import datasets
+
+
+_DESCRIPTION = """\
+A parallel corpus of Ubuntu localization files. Source: https://translations.launchpad.net
+244 languages, 23,988 bitexts
+total number of files: 30,959
+total number of tokens: 29.84M
+total number of sentence fragments: 7.73M
+"""
+_HOMEPAGE_URL = "http://opus.nlpl.eu/Ubuntu.php"
+_CITATION = """\
+@InProceedings{TIEDEMANN12.463,
+  author = {J�rg Tiedemann},
+  title = {Parallel Data, Tools and Interfaces in OPUS},
+  booktitle = {Proceedings of the Eight International Conference on Language Resources and Evaluation (LREC'12)},
+  year = {2012},
+  month = {may},
+  date = {23-25},
+  address = {Istanbul, Turkey},
+  editor = {Nicoletta Calzolari (Conference Chair) and Khalid Choukri and Thierry Declerck and Mehmet Ugur Dogan and Bente Maegaard and Joseph Mariani and Jan Odijk and Stelios Piperidis},
+  publisher = {European Language Resources Association (ELRA)},
+  isbn = {978-2-9517408-7-7},
+  language = {english}
+ }
+"""
+
+_VERSION = "1.0.0"
+_BASE_NAME = "Ubuntu.{}.{}"
+_BASE_URL = "https://object.pouta.csc.fi/OPUS-Ubuntu/v14.10/moses/{}-{}.txt.zip"
+# Please note that only few pairs are shown here. You can use config to generate data for all language pairs
+_LANGUAGE_PAIRS = [
+    ("as", "bs"),
+    ("az", "cs"),
+    ("bg", "de"),
+    ("br", "es_PR"),
+    ("bn", "ga"),
+    ("br", "hi"),
+    ("br", "la"),
+    ("bs", "szl"),
+    ("br", "uz"),
+    ("br", "yi"),
+]
+
+
+class UbuntuConfig(datasets.BuilderConfig):
+    def __init__(self, *args, lang1=None, lang2=None, **kwargs):
+        super().__init__(
+            *args,
+            name=f"{lang1}-{lang2}",
+            **kwargs,
+        )
+        self.lang1 = lang1
+        self.lang2 = lang2
+
+
+class Ubuntu(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class OpusUbuntu(datasets.GeneratorBasedBuilder):
```
this must be the camel case version of the dataset script name `opus_ubuntu`